docs/sources/alerting/unified-alerting/_index.md

```diff
@@ -1,14 +1,14 @@
 +++
-title = "Grafana 8 Alerts"
+title = "Grafana alerts"
 aliases = ["/docs/grafana/latest/alerting/metrics/"]
 weight = 113
 +++
 
-# Overview of Grafana 8 alerting
+# Overview of Grafana alerting
 
-Grafana 8.0 has new and improved alerting that centralizes alerting information in a single, searchable view. It is an [opt-in]({{< relref "./opt-in.md" >}}) feature. We encourage you to create issues in the Grafana GitHub repository for bugs found while testing Grafana 8 alerting. See also, [What's New with Grafana 8 alerting]({{< relref "./difference-old-new.md" >}}).
+Grafana 8.0 has new and improved alerting that centralizes alerting information in a single, searchable view. It is an [opt-in]({{< relref "./opt-in.md" >}}) feature. We encourage you to create issues in the Grafana GitHub repository for bugs found while testing Grafana alerting. See also, [What's New with Grafana alerting]({{< relref "./difference-old-new.md" >}}).
```

Comment:
> It is an [opt-in]({{< relref "./opt-in.md" >}}) feature. 

I think we need to make it more clear that it is opt-in for upgrades that already use legacy. Otherwise, it's the default choice.

Reply:
Clarified.